app-frontend/src/app/components/vectors/vectorNameModal/vectorNameModal.html

```diff
@@ -0,0 +1,35 @@
+<div class="modal-scrollable-body">
+  <div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.handleClose()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <h4 class="modal-title" ng-if="!$ctrl.resolve.project">
+      Create shape
+    </h4>
+  </div>
+
+  <!-- Body for SELECT_FILE -->
+  <div class="modal-body">
+    <div class="content">
+      <h4>Shape name</h4>
+      <input type="text" class="form-control" ng-model="$ctrl.shapeName"></div>
```

Comment:
It would be nice if hitting enter here would submit since there's only a single input.